local-interchain/README.md

```diff
@@ -112,6 +120,21 @@ Here is a base chain template with every feature the configuration accepts. Acco
     "debugging": true,
     "block_time": "500ms",
     "encoding-options": ["juno"],
+    "host_port_override": {
+        "26657": "26657",
+        "1317": "1320",
```

Comment:
nit: it's just a template, but is 1320 a typo?

Reply:
This was to show that internal 1317 maps to 1320. Agree it is confusing so changed to add a documentation note instead and use 1317:1317 mapping

14b34db35c3c201088b433b667645d046881c6f4